src/ebmc/transition_system.cpp

```diff
@@ -92,6 +92,10 @@ int preprocess(const cmdlinet &cmdline, message_handlert &message_handler)
     return 1;
   }
 
+  // do -I
+  if(cmdline.isset('I'))
+    config.verilog.include_paths = cmdline.get_values('I');
+
```

Comment:
Given that it seemed surprising that no one noticed this didn't work: could you please add a test?